postgrest.cabal

```diff
@@ -122,6 +130,7 @@ library
                      , PostgREST.RangeQuery
                      , PostgREST.ApiRequest
                      , PostgREST.Types
+                     , PostgREST.ApiSpec
```

Comment:
How about Swagger, ApiSpec implies like this is the only description that we want to support